<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KOutOrder</class>
 <widget class="QWidget" name="KOutOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KOutOrder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="3,4">
     <item>
      <widget class="QTextBrowser" name="textBrowserOrderPageOrderInfo"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1,6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,0,7">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>类别：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxOrderPageCategory">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelOrderPageId">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,9">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>名称：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxOrderPageName">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4,1">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>单价：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditOrderPageUnitPrice">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border:none;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>￥</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4,1">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>库存：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditOrderPageInventory">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border:none;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>件</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>数量：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxOrderPageNum">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border:none;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,4,1">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>总价：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditOrderPageTotalPrice">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border:none;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>￥</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
         <item>
          <widget class="QPushButton" name="btnOrderPagePicture">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,1">
           <item>
            <widget class="QTextEdit" name="textEditOrderPageInfo"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnOrderPageSellOff">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>60</height>
              </size>
             </property>
             <property name="text">
              <string>出售</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnOrderPageUnderOrder">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>60</height>
              </size>
             </property>
             <property name="text">
              <string>下单</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
